x/xos/file.go

```diff
@@ -0,0 +1,25 @@
+package xos
+
+import "os"
+
+// Remove removes all given named file or directory.
+func Remove(names ...string) error {
+	var err error
+	for _, name := range names {
+		if err1 := os.Remove(name); err == nil {
+			err = err1
+		}
+	}
+	return err
+}
+
+// RemoveAll removes all given paht and any children it contains.
+func RemoveAll(paths ...string) error {
```

Comment:
can be renamed as RemoveRecursive, if you wish

Reply:
RemoveAll is fine for me

Reply:
Leaving RemoveAll